### PR TITLE
Update layout label

### PR DESCRIPTION
**Pull request by sofiamatulis**
Update layout label for global spacing setting

### CODE REVIEW

**Comment by sofiamatulis** [comment]
@melissaperreault @ludoboludo @eugenekasimov If this looks good ill request translations 🙌

**Review by ludoboludo — APPROVED** [review]
Sounds good 👍

**Review by eugenekasimov — APPROVED** [review]
🚀

File: locales/fr.schema.json [hunk 1]
```diff
@@ -157,7 +157,7 @@
           "label": "Largeur de la page"
         },
         "spacing_sections": {
-          "label": "Espace vertical entre les sections"
+          "label": "Espace entre les sections de modèle"
```
[the remark this answers is not in the document]
Comment (sofiamatulis) [reply locales/fr.schema.json:160]: Any suggestions of what we do in this scenario? 👀 cc @melissaperreault

[the remark this answers is not in the document]
Comment (melissaperreault) [reply locales/fr.schema.json:160]: For context; from what I can see from the translations related to the word "Template" it should be `Espace entre les sections **du** modèle.`

**Pull request merged**